implementations/lighthouse/app/Models/Product.php

```diff
@@ -14,6 +15,8 @@
  * @property string|null $package
  * @property string|null $variation
  * @property array{size: string|null, weight: float|null}|null $dimensions
+ * @property string|null $notes
+ * @property array{array{study:array{caseNumber: string, description: string|null}, outcome: string|null}} $research
```

Comment:
This should be `@property-read EloquentCollection<int, ProductResearch> $research and moved down to the section `Relations`.

Also add an import `use Illuminate\Database\Eloquent\Collection as EloquentCollection;` up top.